configs/settings/base.py

```diff
@@ -29,6 +29,7 @@
     'rest_framework',
     'rest_framework_simplejwt',
     'accounts.apps.AccountsConfig',
+    'accounts.apps.PaysConfig',
```

Comment:
❗ 경로가 잘못되었어요! pays 폴더의 apps 파일의 PaysConfig 클래스를 지정해야 합니다.
```suggestion
    'pays.apps.PaysConfig',
```